configure.ac

```diff
@@ -286,8 +286,18 @@ yes_no_try "$enable_legacy_links" "no"
 enable_legacy_links=$?
 AM_CONDITIONAL([BUILD_LEGACY_LINKS], [test $enable_legacy_links -ne $DISABLED])
 
-dnl --with-* options: external software support, and custom locations
+dnl --disable-nls option 
+AM_GNU_GETTEXT([external])
+AM_GNU_GETTEXT_VERSION(0.18)
+AM_CONDITIONAL([USE_NLS], [test "x${USE_NLS}" = "xyes"])
```

Comment:
I don't think there are any Makefiles that need USE_NLS anymore, so we can drop this line